### PR TITLE
Include instructions for one-line call to clang w Enzyme as a plugin

### PR DESCRIPTION
**Pull request by ludgerpaehler**
Trying to point out "Getting Started"'s nature of breaking down the individual steps involved in compilation, and being able to summarise these steps into a single compilation call.

Questions left open for the moment are:
- Are we able to highlight the single-line call with a highlighting box or similar, e.g. by injecting HTML code?
- Should this have its own separate sub-header?

### CODE REVIEW

File: content/getting_started/UsingEnzyme.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ draft: false
 weight: 10
 ---
 
+---
```
Comment (wsmoses) [inline content/getting_started/UsingEnzyme.md:8]: This shouldn't be repeated?

Comment (ludgerpaehler) [reply content/getting_started/UsingEnzyme.md:8]: Yes, this should absolutely not get repeated!

File: content/getting_started/UsingEnzyme.md [hunk 2]
```diff
@@ -120,9 +127,15 @@ We can then compile this to a final binary as follows:
...
+For ease, we could combine the final optimization and binary execution into one command as follows.
+```sh
+clang output.ll -O2 -o a.exe
+```
+
+These individual steps can be summarised into a single compilation step, where we load Enzyme as a plugin to our compiler clang, and perform the differentiation on our test code.
+
 ```sh
-clang output.ll -O3 -o a.exe
+clang test.c -O2 -Xclang -load -Xclang /path/to/Enzyme/enzyme/build/Enzyme/ClangEnzyme-<VERSION>.so -fno-vectorize -fno-slp-vectorize -fno-unroll-loops -o a.exe
```
Comment (wsmoses) [inline content/getting_started/UsingEnzyme.md:138]: I might rotate this and start by describing the clang path, and then discussing the internals as what's happening

File: content/getting_started/UsingEnzyme.md (hunk 1 above)
Comment (ludgerpaehler) [reply content/getting_started/UsingEnzyme.md]: Solved it with the patch below

File: content/getting_started/UsingEnzyme.md (hunk 2 above)
Comment (ludgerpaehler) [reply content/getting_started/UsingEnzyme.md:131]: Yes, that would definitely be an option. I think we should just make it more apparent that the steps explained as of now are basically a breakdown of the compilation steps when Enzyme is used as a compiler plugin, as in the call on line 138.

**Comment by ludgerpaehler** [comment]
Closed due to being included in the docs pull-request, with the above comments worked into the new ``first-derivative`` page.

**Pull request closed without merging**